### PR TITLE
transfer from bitbucket

### PR DESCRIPTION
**Pull request by arnaudbore**
Before merging
- [x] add the save/load kernels

Need https://github.com/scilus/scilpy/pull/342

### CODE REVIEW

**Review by mdesco — COMMENTED** [review]
Can you change output directory of ComputedKernels to be inside the results directory with the other files please. 

Also, my last step dti_metrics crashes.

**Review by mdesco — COMMENTED** [review]
Before we merge this, can you add the 
 --b_thr B_THR to the options of the nextflow and scil_compute_freewater.py command please

Same for NODDI

**Comment by arnaudbore** [comment]
> Before we merge this, can you add the
> --b_thr B_THR to the options of the nextflow and scil_compute_freewater.py command please
> 
> Same for NODDI

Done

File: USAGE
Comment (mdesco) [inline USAGE]: Can you reformat exactly as NODDI flow please for uniformity.

If files come out of the Tractoflow pipeline, you should take the following source:
- dwi.nii.gz: from Resample_DWI
- brain_mask.nii.gz: from Resample_B0
- bval/bvec: from Eddy_Topup (or Eddy)

**Review by mdesco — APPROVED** [review]
This can be merged right away after little edit of the USAGE.

File: nextflow.config
Comment (GuillaumeTh) [inline nextflow.config:2]: Do we add a retry strategy ?

**Pull request merged**
